### PR TITLE
Additional changes to code-xy pages to improve readability

### PR DESCRIPTION
**Pull request by aileenrose**
I updated code pages to improve readability for new visitors to the pages.  
On the code review page, I included an excerpt of the GitLab video which focuses on code discovery and code review.
https://deploy-preview-1055--sourcegraph.netlify.app/product/code-review
https://deploy-preview-1055--sourcegraph.netlify.app/product/code-discovery
https://deploy-preview-1055--sourcegraph.netlify.app/product/code-intelligence

Note: I did not remove the CTA demo request from the code change mgmt page since this feature is in beta.

### CODE REVIEW

File: website/src/pages/product/code-review.tsx [hunk 1]
```diff
@@ -13,34 +13,24 @@ import { Jumbotron } from '../../components/Jumbotron'
...
-                    <RequestDemoAction className="mt-3" />
-                    <ViewDeveloperDocumentationAction
-                        className="text-light mt-2"
-                        url="https://docs.sourcegraph.com/#quickstart"
-                    >
-                        Documentation &amp; self-service install
-                    </ViewDeveloperDocumentationAction>
-                </div>
-            }
+            description="Make your code reviews fast and thorough so you can catch more bugs before they're deployed to production."
```
Comment (adaaaam) [inline website/src/pages/product/code-review.tsx:25]:
```suggestion
            description="Make code reviews fast and thorough to catch more bugs before they're deployed to production."
```

File: website/src/pages/product/code-review.tsx [hunk 2]
```diff
@@ -74,14 +64,14 @@ export default ((props: any) => (
...
                     <div className="col-lg-10 text-center">
-                        <img src="/external-logos/lyft-logo.svg" width="100px" />
+                        <img src="/external-logos/gitlab-logo.svg" width="125px" />
                         <blockquote className="blockquote case-studies__quote case-studies__quote--in-content">
                             <p>
-                                Sourcegraph [code search] … has made me insanely more productive and efficient at
-                                writing code here. I’m able to understand and deeply dive through all of our
-                                microservices and get my work done really fast.
+                                The Sourcegraph integration [with GitLab] also adds inline code intelligence when you
+                                review code in merge requests (MRs) which helps you speed up the review cycle and catch
```
Comment (adaaaam) [inline website/src/pages/product/code-review.tsx:71]:
```suggestion
                                review code in merge requests which helps you speed up the review cycle and catch
```

**Comment by aileenrose** [comment]
I prefer to have the GitLab video first since Sid walks through the demo.  The other video does not include sound.

**Comment by adaaaam** [comment]
I think the other demo is better since it's more general but we can go ahead with this for now.

**Pull request merged**